### DISCUSSION TITLE
Unable to update Company info

### DISCUSSION THREAD

**Discussion by blackyzero** [saleor/saleor#9896] [General] [answered]
Hello,

When trying to update company info, i get following error from Saleor console.
How could i fix it?

`==> store-saleor-error.log <==
[01/Jun/2022 19:19:24] "OPTIONS /graphql/ HTTP/1.1" 200 0
[01/Jun/2022 19:19:24] "POST /graphql/ HTTP/1.1" 200 761
[01/Jun/2022 19:19:24] "POST /graphql/ HTTP/1.1" 200 18839
[01/Jun/2022 19:19:24] "POST /graphql/ HTTP/1.1" 200 86342
[01/Jun/2022 19:19:44] "OPTIONS /graphql/ HTTP/1.1" 200 0
INFO saleor.graphql.errors.handled A query had an error [PID:3701:Thread-154]
graphql.error.base.GraphQLError: Variable "$isCloudInstance" is never used in operation "ShopSettingsUpdate".
WARNING django.request Bad Request: /graphql/ [PID:3701:Thread-154]
[01/Jun/2022 19:19:45] "POST /graphql/ HTTP/1.1" 400 417
`

Thank you.

**Comment by karolkielecki** [comment]
Hi @blackyzero, could you please provide the steps to reproduce the issue (ideally with some screenshots), and which version are you using?

**Comment by blackyzero** [comment]
Hello @karolkielecki 

I am running dashboard v3.4.0, core v3.4.1

Step to reproduce:

In Configuration - Site Settings - Company Info, input company info and click Save, then error output from saleor console log.
```
[02/Jun/2022 17:05:39] "OPTIONS /graphql/ HTTP/1.1" 200 0
INFO saleor.graphql.errors.handled A query had an error [PID:598:Thread-555]
graphql.error.base.GraphQLError: Variable "$isCloudInstance" is never used in operation "ShopSettingsUpdate".
WARNING django.request Bad Request: /graphql/ [PID:598:Thread-555]
[02/Jun/2022 17:05:39] "POST /graphql/ HTTP/1.1" 400 417
```

And this issue is showing exactly from [demo.](https://demo.saleor.io/dashboard/configuration/)
Please refer attached screenshot for more detail.

Thank you.

![Screen Shot 2022-06-02 at 1 10 20 PM](https://user-images.githubusercontent.com/4286063/171685941-5c481ff2-1dd1-4bda-90aa-b7ad1e0dbffc.png)

**Comment by timuric** [comment, marked as answer]
@blackyzero this is fixed in 
```
dashboard v3.4.1
core v3.4.5
```

**Comment by blackyzero** [comment]
great. Thanks @timuric

**Comment by nadahmed** [comment]
Having the same issue on Saleor core: 3.6.3 and Saleor Dashboard: 3.6.0
